Lesson_13/homework_13_csv.py

```diff
@@ -0,0 +1,34 @@
+import csv
+
+with open('r-m-c.csv', newline='', encoding='utf-8') as first_file:
```

Comment:
All this code needs to be stored in separate functions based on logical portion. It can be developed in your way - but in best practice is better to store it in separate functions.